bleachbit/GuiPreferences.py

```diff
@@ -178,6 +178,11 @@ def __general_page(self):
             'toggled', self.__toggle_callback, 'delete_confirmation')
         vbox.pack_start(cb_popup, False)
 
+        # Use base 1000 oder 1024?
+        cb_units_iec = gtk.CheckButton(_("Use IEC sizes (1 KiB = 1024 bytes) instead of SI (1 kB = 1000 bytes)"))
```

Comment:
I know so far all the other options in this menu use CheckButton, but maybe this choice would be more easier for the user to understand this choice as a radio button? Check boxes are usually used to indicate something is enabled or disabled, while radio boxes are a set of mutually-exclusive options. Conceivably there could be a third option other than SI or IEC. Or another way of thinking about it that "IEC enabled and SI disabled" could be "SI enabled and IEC disabled," so the enabled/disabled is not intuitive.